spec/classes/puppet_server_puppetserver_spec.rb

```diff
@@ -468,7 +486,30 @@
           end
           context 'with server_trusted_agents' do
             let(:params) { super().merge(server_puppetserver_trusted_agents: ['jenkins', 'octocatalog-diff']) }
-            it { should contain_file(auth_conf).with_content(%r{^(\ *)allow: \["jenkins", "octocatalog-diff"\]\n(\ *)sort-order: 500\n(\ *)name: "puppetlabs v4 catalog for services"}) }
+            it do
+              content = catalogue.resource('file', auth_conf).send(:parameters)[:content]
```

Comment:
Could you apply the same let trick here as above?

Reply:
oops.  Of course.